### PR TITLE
MB-5053 update devlocal readme instructions:

### PR DESCRIPTION
**Pull request by kilbergr**
## Description

https://dp3.atlassian.net/browse/MB-5053

This PR should be the finishing touch on moving all devs over to using devlocal in govcloud. It updates the readme with changed information.

## Reviewer Notes

Going through with a new app eng to make sure things work as expected. Would like an established app-eng to also try (perhaps one of those who onboarded while we were transitioning or a long-term team member).

## Setup

Please follow the readme! 

## Code Review Verification Steps

* [x] Request review from a member of a different team.
* [x] Have the Jira acceptance criteria been met for this change?

## References

* [Jira story](https://dp3.atlassian.net/browse/MB-5053) for this change

### CODE REVIEW

**Review by travelar — APPROVED** [review]

**Review by esacteksab — APPROVED** [review]

**Review by jenniferlynparsons — APPROVED** [review]
LGTM! Along with https://github.com/transcom/mymove/pull/5420 this allowed me to successfully get my dev env set up.

**Review by mdawn — APPROVED** [review]
🍾

**Review by gidjin — APPROVED** [review]
I was able to access the variables as documented, and I was able to update the few vars for A-team using the write command! I don't see it in the PR but I see that `.envrc.chamber.template` is updated too!

File: README.md [hunk 1]
```diff
@@ -234,7 +234,7 @@ Run `direnv allow` to load up the `.envrc` file. It should complain that you hav
 
 You can add a `.envrc.local` file. One way to do this is using the [chamber tool](https://github.com/segmentio/chamber) to read secrets from AWS vault. To use the AWS vault you will need to follow the [instructions to set up AWS first](#setup-aws-services-optional).
```
Comment (duncan-truss) [inline README.md:235]: In the link to `instructions to set up AWS first` we're still linking to the commercial repo for setting up new aws users.  Did we want to update that to point to the [gov runbook](https://github.com/transcom/transcom-infrasec-gov/blob/master/docs/runbook/0001-aws-organization-authentication.md) too?

**Review by duncan-truss — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (kilbergr) [reply README.md:235]: Great catch!! Thank you

**Review by mboren — APPROVED** [review]
This covers all of the issues I ran into during setup 👍

**Pull request merged**